r/src/altrep.cpp

```diff
@@ -87,13 +87,11 @@ class ArrowAltrepData {
 
   const std::shared_ptr<ChunkedArray>& chunked_array() { return chunked_array_; }
 
-  arrow::internal::ChunkLocation locate(int64_t index) {
-    return resolver_.Resolve(index);
-  }
+  arrow::ChunkLocation locate(int64_t index) { return resolver_.Resolve(index); }
 
  private:
   std::shared_ptr<ChunkedArray> chunked_array_;
-  arrow::internal::ChunkResolver resolver_;
+  arrow::ChunkResolver resolver_;
```

Comment:
I created PR https://github.com/anjakefala/arrow/pull/1 which should fix https://github.com/apache/arrow/actions/runs/11392461120/job/31698584591?pr=44357.

Reply:
Thank you!